frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -41,10 +37,9 @@ const validateVm: Validator = (field, options) => {
   const isCreateTemplate = iGetCommonData(state, id, VMWizardProps.isCreateTemplate);
 
   const entities = isCreateTemplate
-    ? concatImmutableLists(
-        iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates),
-        iGetLoadedCommonData(state, id, VMWizardProps.userTemplates),
-      )
+    ? concatImmutableLists(iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates), [
```

Comment:
this will not validate properly we need another prop (`userTemplates`) just for the templates flow - it should be needed just for this validation, if I am not mistaken.

also commonTemplates are not needed here

Reply:
Please do not revert the changes you have doen already for this.